.github/stale.yml

```diff
@@ -0,0 +1,9 @@
+onlyLabels: ['Status: Waiting feedback']
+daysUntilStale: 14
+daysUntilClose: 7
+staleLabel: 'Won''t Fix'
```

Comment:
I'd rather use a `stale` label here to identify a stale issue awaiting feedback than `Won't fix`, which means something else.

Reply:
Fine by me